File: 151003/Shafarenko/lab5/redis/Publisher/Repository/Implementation/MarkerRepository.cs
using Publisher.Entity.Db;
using Publisher.Repository.Implementation.Common;
using Publisher.Repository.Interface;
using Publisher.Storage.Common;

namespace Publisher.Repository.Implementation
{
    public class MarkerRepository(DbStorage context) : AbstractCrudRepository<Marker>(context), IMarkerRepository
    {
    }
}

File: 151002/Naftolski/Lab5/MessageApp/publisher/src/main/java/by/bsuir/publisher/dao/repository/MarkerRepository.java
package by.bsuir.publisher.dao.repository;

import by.bsuir.publisher.model.entity.Marker;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MarkerRepository extends JpaRepository<Marker, Long> {

}
